cluster/images/provider-kafka-controller/Dockerfile

```
@@ -1,3 +1,10 @@
-FROM BASEIMAGE
+FROM gcr.io/distroless/static@sha256:d2b0ec3141031720cf5eedef3493b8e129bc91935a43b50562fbe5429878d96b
 
-COPY package.yaml .
+ARG TARGETOS
+ARG TARGETARCH
+
+ADD bin/$TARGETOS\_$TARGETARCH/provider /usr/local/bin/crossplane-kafka-provider
+
+EXPOSE 8080
+USER 1001
```

Last update here, sets uid to nonroot user in distroless base 👍🏻  
```suggestion
USER 65532
```

---

Updated to 65532